js/index.js

```diff
@@ -216,12 +393,147 @@ function renderProjectsData(data){
       projectToolsContainer.appendChild(projectToolTitle);
       projectToolsContainer.appendChild(projectToolsList);
 
-      //Increment the arrays' indexes.
-      arrayCounter++;
     }
   }
 }
 
+function showEducation(){
+  //1. Setting up education data:
+  //1.1 Variables:
+  const educationData = [
+    {
+      title: "Computer Science Engineer",
+      date: "January 2016 - May 2021",
+      image: "./images/education/College-Degree.png",
```

Comment:
One more case correction to make, this references `/education/`, the actual directory appears to be upper cased `/Education/`

Reply:
I will take a look into that right now, thanks!